akka-docs/src/main/paradox/persistence-typed.md

```diff
@@ -1,32 +1,188 @@
 # Persistence 
 
-TODO
-
-## Dependency
-
-sbt
-:   @@@vars
-    ```
-    "com.typesafe.akka" %% "akka-persistence-typed" % "$akka.version$"
-    ```
-    @@@
-
-Gradle
-:   @@@vars
-    ```
-    dependencies {
-      compile group: 'com.typesafe.akka', name: 'akka-persistence-typed_2.11', version: '$akka.version$'
-    }
-    ```
-    @@@
-
-Maven
-:   @@@vars
-    ```
-    <dependency>
-      <groupId>com.typesafe.akka</groupId>
-      <artifactId>akka-persistence-typed_$scala.binary_version$</artifactId>
-      <version>$akka.version$</version>
-    </dependency>
-    ```
-    @@@
+@@@ warning
+
+This module is currently marked as @ref:[may change](common/may-change.md) in the sense
+  of being the subject of active research. This means that API or semantics can
+  change without warning or deprecation period and it is not recommended to use
+  this module in production just yet—you have been warned.
+  
+@@@
+
+@@@ warning
+
+This module only has a Scala DSL. See [#24193](https://github.com/akka/akka/issues/24193) 
+to track progress and to contribute to the Java DSL.
+  
+@@@
+
+To use typed persistence add the following dependency:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-persistence-typed_2.11
```

Comment:
should we start promoting 2.12 for java?
I think we can replace all `_2.11`

Reply:
👍 we can do this en massed across all projects in 2018 I think 👍